### PR TITLE
fix: Windows 3.13 stable ABI

### PR DESCRIPTION
**Pull request by LecrisUT**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -321,6 +324,7 @@ jobs:
       fail-fast: false
       matrix:
         runs-on: [ubuntu-latest, macos-14, windows-latest]
+        python-version: ["3.11", "3.13"]
```
Comment (henryiii) [inline .github/workflows/ci.yml:327]: We shouldn't double up the docs, it's causing timeouts when checking URLs. Three's already pushing it sometimes.

Comment (LecrisUT) [reply .github/workflows/ci.yml]: Yeah, just opened this to try and reproduce the related issue, feel free to rebase accordingly. Btw why not use `3.x` in those workflows?

Also it didn't seem to have worked, any notes on why it didn't use 3.13 there?

Comment (henryiii) [reply .github/workflows/ci.yml]: IIRC the nox action always uses the same Python to run itself, regardless of what you ask for (either 3.11 or 3.12, don't remember). And then if you don't specify a Python, it uses the one it ran with.

Comment (henryiii) [reply .github/workflows/ci.yml]: > The compose Fedora-41 is not available in the public Testing

Do you know what that failure is?

Comment (LecrisUT) [reply .github/workflows/ci.yml]: > Do you know what that failure is?

Yeah, fedora branched a few days ago, and `testing-farm` didn't catch up yet. I've reported this morning and they should be fixing it by tomorrow.

> And then if you don't specify a Python, it uses the one it ran with.

I guess you mean if you don't specify the Python version inside the nox file or cli?

Comment (henryiii) [reply .github/workflows/ci.yml]: Yes, exactly.

File: tests/packages/abi3_pyproject_ext/CMakeLists.txt [hunk 2]
```diff
@@ -11,6 +11,10 @@ find_package(
   REQUIRED)
 
 if(NOT "${SKBUILD_SABI_COMPONENT}" STREQUAL "")
+  include(CMakePrintHelpers)
+  cmake_print_variables(
+    Python_SABI_LIBRARY Python_SOSABI Python_SABI_LIBRARIES
+    Python_SABI_LIBRARY_DIRS Python_RUNTIME_SABI_LIBRARY_DIRS)
```
Comment (henryiii) [inline tests/packages/abi3_pyproject_ext/CMakeLists.txt:17]:
```suggestion
```

**Pull request merged**